bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaProcessorHandler.java

```diff
@@ -325,9 +281,10 @@ private void handleStatusUpdate(EmotivaUdpResponse emotivaUdpResponse) {
             return;
         }
 
-        if (object instanceof EmotivaAckDTO answerDto) {
+        if (object instanceof EmotivaAckDTO) {
             // Currently not supported to revert a failed command update, just used for logging for now.
-            logger.trace("Processing received '{}' with '{}'", EmotivaAckDTO.class.getSimpleName(), answerDto);
+            logger.trace("Processing received '{}' with '{}'", EmotivaAckDTO.class.getSimpleName(),
+                    emotivaUdpResponse.answer());
 
```

Comment:
SAT Error
```suggestion
```

Reply:
This was the casue of the conflict, already removed on main.